tests/console/orphaned_packages_check.pm

```diff
@@ -21,7 +21,7 @@ use version_utils 'is_upgrade';
 
 sub run {
     select_console 'root-console';
-    my $cmd = 'zypper pa --orphaned | grep -v "\(release-DVD\|release-dvd\|openSUSE-release\|skelcd\)" | (! grep "@System")';
+    my $cmd = 'zypper pa --orphaned | grep -v "\(release-DVD\|release-dvd\|openSUSE-release\|skelcd\)" | (! grep "@System") > /tmp/orphaned.log';
```

Comment:
We'd better to add comments here for the reason that we need the log.

Reply:
I think "tee" would be better than ">" to see the output in the test progress as well.

Reply:
> I think "tee" would be better than ">" to see the output in the test progress as well.

If use 'tee', even has orphaned packages, it cannot return error. Please check 
http://10.161.8.44/tests/1008#step/orphaned_packages_check/9